### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by BouaikIlham**
### Project requirements
- [ ]  First, I choose an API this is it ( https://api.covid19api.com/summary)
- [ ]  I follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
     - Colors (select one main color for your website).
     - Typography: font face, size and weight.
     - Layout: composition and space between elements.
 - [ ] I started with the home page : 
      - When the page loads, the web app shows the list of categories that could be filtered out by some parameter
      - Along with the category name, you will display some numeric values per category that come from the API.
      - When the user clicks (or taps) on a category item, the application navigates to the details page.
  - [ ] Next the details page:
      - On the details page, the webapp retrieves data from the API to show detailed data in the category.
      - When the user clicks on the "Back" button (<), the user navigates to the home page.
  - [ ] Create unit tests for pure functions
  - [ ] The project deployed and accessible online
  ### Project documentation
   - [ ] Vidio recording in Loom is here => [link](https://www.loom.com/share/aaaaec08a45642b4a1a5107b9a02e3fd)

### CODE REVIEW

File: src/tests/Navbar.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          <Navbar />
+        </BrowserRouter>
+        ,
+      </Provider>,
+    );
+
+    expect(
+      screen.getByRole('heading', { name: /COVID METRICS/i }),
+    ).toBeInTheDocument();
+  });
```
Comment (RolandM99) [inline src/tests/Navbar.test.js]: - [ ] Make sure that all your test cases are correct. For now, this test failed, seems that the expectation is wrong. Kindly solve this issue by changing the name with the right input. ( It should be "2022" for your case).
image

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi  @BouaikIlham, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Good job by creating a loom video for your project presentation. It seems that you forgot to test your microphone before recording your video presentation. it's muted, and I can't listen to what you're explaining. Kindly, record a new one ,with voice and also by the way don't forget to highlight  a piece of the code that you find interesting or you're proud of making. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/Navbar.test.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ describe('Navbar tests', () => {
     );
 
     expect(
-      screen.getByRole('heading', { name: /COVID METRICS/i }),
+      screen.getByRole('heading', { name: /2022/i }),
```
Comment (mateo951) [inline src/tests/Navbar.test.js:24]: Nice ✔

**Review by mateo951 — APPROVED** [review]
### Hi @BouaikIlham, 

Well done making the changes required by the previous reviewer ✔
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it 🔥 

Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**